prime-router/src/main/kotlin/LookupTable.kt

```diff
@@ -122,7 +122,7 @@ class LookupTable(
      * Get the set of distinct values in a column in the table.
      */
     fun getDistinctValuesInColumn(selectColumn: String): Set<String> {
-        val selectColumnNumber = headerIndex[selectColumn.toLowerCase()] ?: return emptySet()
+        val selectColumnNumber = headerIndex[selectColumn.lowercase()] ?: return emptySet()
```

Comment:
Do you know why my IDE (intellij) doesn't accept the method 'lowercase'? It's weird that the one it accepts is the one you replaced.

Reply:
> Do you know why my IDE (intellij) doesn't accept the method 'lowercase'? It's weird that the one it accepts is the one you replaced.

That happened to me.  You need to make sure to install the Kotlin 1.5.0 plugin, which may still be in beta or unnoficial.